cmd/lifecycle/exporter.go

```diff
@@ -249,7 +268,7 @@ func (ea exportArgs) export(group lifecycle.BuildpackGroup, cacheStore lifecycle
 	return nil
 }
 
-func initDaemonImage(imagName string, runImageRef string, analyzedMD lifecycle.AnalyzedMetadata, launchCacheDir string, docker client.CommonAPIClient) (imgutil.Image, string, error) {
+func (ea exportArgs) initDaemonImage(runImageRef string, analyzedMD lifecycle.AnalyzedMetadata) (imgutil.Image, string, error) {
```

Comment:
```suggestion
func (ea exportArgs) initDaemonAppImage(runImageRef string, analyzedMD lifecycle.AnalyzedMetadata) (imgutil.Image, string, error) {
```
While we are here? Just to make it clear that it isn't completely generic.